charts/common/values.yaml

```diff
@@ -0,0 +1,600 @@
+## Default values for common.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## The following values are mandatory when using "helm lint --strict"
+#global:
+#  name: ""
+#  namePrefix: ""
+#  nameSuffix: ""
+#  fullnamePrefix: ""
+#  fullnameSuffix: ""
+#namePrefix: ""
+#nameSuffix: ""
+#fullnamePrefix: ""
+#fullnameSuffix: ""
+#nameOverride: ""
+#fullnameOverride: ""
+
+# defaultTag is used when no imageTag is specified in .Values.components.*.controller.image.tag
+defaultTag: latest
+
+# timezone to set as environment variable 'TZ' in each pod. Comment out for using default ("Europe/Zurich")
+#timezone: "Europe/Zurich"
+
+# secrets contains the pullSecret for the container registry to pull images.
+# In a Bedag namespace this value does not have to be set for Dockerhub or the Bedag registry
+secrets:
+  data:
+    registry:
+      pullSecret: ""
+
+# start common.networkpolicy
+# networkpolicy restricts all access between the pods. To configure allowed connections, go to components.*.networkpolicy.podSelector
+networkpolicy:
+  # deploy has to be set to true for rendering to be applied
+  deploy: true
+# end common.networkpolicy
+
+# start common.ingress
+# ingress is used for access from outside of kubernetes
+ingress:
+  # deploy has to be set to true for rendering to be applied
+  deploy: false
+  # IngressClassName, defines the class of the ingress controller. For Bedag classes see https://confluence.se.bedag.ch/display/KUBERNETES/Service+Katalog
```

Comment:
remove internal links (multiple times in this file)

Reply:
done